app/src/main/res/layout/fragment_dogs_detail.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/LinerLayout1"
```

Comment:
CamelCase in xml. Should be this way **ll1** or **linear_layout**. Also name is none informative.

Reply:
ready